File: CS502024/sentimental-mario-more/mario.py
import cs50

while True:
    height = cs50.get_int("Height: ")
    if height > 0 and height < 9:
        break
for i in range(1, height+1):
    for j in range(height-i):
        print(" ", end="")
    for k in range(i):
        print("#", end="")
    print("  ", end="")
    for l in range(i):
        print("#", end="")
    print()

File: CS502024/mario/more/mario.c
#include <stdio.h>
#include <cs50.h>

int main() {
    int height;
    do {
        height = get_int("Height: ");
    } while(height < 1 || height > 8);
    for(int i = 0; i < height; i++){
        for(int j = 0; j < height-1-i; j++) {
            printf(" ");
        }
        for(int k = 0; k <= i; k++) {
            printf("#");
        }
        printf("  ");
        for(int l = 0; l <= i; l++) {
            printf("#");
        }
        printf("\n");
    }
    return 0;
}
